manim/utils/config_utils.py

```diff
@@ -130,23 +133,46 @@ def _parse_cli(arg_list, input=True):
         description="Animation engine for explanatory math videos",
         epilog="Made with <3 by the manim community devs",
     )
+    min_argvs = 4 if "py" in sys.argv[0] else 2
+    if input and (len(sys.argv) > min_argvs-1 # If "manim" is not the only command
+        and any(a == item for a in sys.argv for item in NON_ANIM_UTILS) #non-anim exists
+        or len(sys.argv) == min_argvs-1):
```

Comment:
Suggestion: Put all of this into a separate function for readability. Kind of like how it was done for `_from_command_line()`.

Reply:
Comment: yes, this is hacky, but as long as it works I think it's fine. Perhaps open an issue after merging this flagging that this should/cloud be improved.

Reply:
> Suggestion: Put all of this into a separate function for readability.

I cannot thank you enough for pushing me to do this. I've managed to get rid of an extra `if` statement, and the readability is improved by like 10 times!